src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/releaseChart.tsx

```diff
@@ -39,7 +39,7 @@ class ReleaseChart extends React.Component<Props> {
     const {yAxis} = this.props;
     switch (yAxis) {
       case 'sessionDuration':
-        return typeof value === 'number' ? getExactDuration(value, true) : value;
+        return typeof value === 'number' ? getExactDuration(value, true) : '-';
```

Comment:
Should we use \u2013 here (n-dash) or \u2015 (m-dash) here?

Reply:
Good idea